docs/plans/2026-01-01-batch-commands-and-skills-design.md

```
@@ -0,0 +1,270 @@
+      "name": "ableton-mcp",
+      "description": "AI-assisted music production skills for Ableton Live",
+      "version": "1.0.0",
+      "source": "./"
+    }
+  ]
+}
+```
+
+### Core Skills
+
+**1. create-track-template**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)